### PR TITLE
Don't attempt to pull docker images on relaunch

### PR DESCRIPTION
**Pull request by marcosdiez**
Hi again!
If we are relaunching a cluster, we already have pulled the images (and if they ever got deleted, it's not the end of the world). That saves startup time!

The fact is, since I am physically far from the k8s.gcr.io servers, a simple `time sudo kubeadm config images pull --config "/var/lib/kubeadm.yaml"` takes around 30 seconds if the images are already there.

From an AWS server in the US the same command takes only 5 seconds, but, well, it's still a performance improvement without any downside :) 

Please don't think that I have the worse connection on earth and hence I am an edge case. Quite the opposite. I am just physically far for `k8s.gcr.io` and therefore the latency is high`.

### CODE REVIEW

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by RA489** [comment]
@marcosdiez please squash the commits to one

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -574,6 +570,11 @@ func bootstrapCluster(bs bootstrapper.Bootstrapper, r cruntime.Manager, runner b
 			exit.WithLogEntries("Error restarting cluster", err, logs.FindProblems(r, bs, runner))
 		}
 		return
+	} else {
+		console.OutStyle("pulling", "Pulling images required by Kubernetes %s ...", kc.KubernetesVersion)
```
Comment (tstromberg) [inline cmd/minikube/cmd/start.go]: I like this change - but there is a bit more subtlety here that is missing. This will also need to handle the case where the pre-existing cluster is a different version of Kubernetes.

**Comment by tstromberg** [comment]
Any idea on how much time this saves? I want to make sure the added complexity is worth it.

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (marcosdiez) [reply cmd/minikube/cmd/start.go]: Here in Brazil it really saves me 30 seconds. So I will later send you will a PR to check if the server is being upgraded or not.

I read that you aim to make minikube launch in less than a minute. 

With this patch (and all the others you have merged), here it loads in 53 seconds. How long does it take to load for you ?

**Comment by marcosdiez** [comment]
@tstromberg Implementing the check was easier than I thought. And I only had to add 4 lines, one of them being blank. I also rebased, as you requested.

**Comment by tstromberg** [comment]
@minikube-bot OK to test
